silver/models/documents.py

```diff
@@ -0,0 +1,706 @@
+# Copyright (c) 2016 Presslabs SRL
```

Comment:
Why not using a `documents` module which contains multiple files and each file a different model?